scripts/train_lm.py

```diff
@@ -63,6 +68,8 @@
                     help="Directory containing the training text.")
 parser.add_argument("order",
                     help="Order of N-gram model to be trained.")
+parser.add_argument("work_dir",
+                    help="Working directory for building the language model.")
 parser.add_argument("lm_dir",
```

Comment:
Because some Kaldi scripts using this have already been checked in, 'lm_dir' now needs to be an optional argument.
Make clear that this will default to a subdirectory of work_dir if not provided, with name depending on the n-gram order and vocabulary size.